helidon-cli/impl/pom.xml

```diff
@@ -37,6 +37,16 @@
             <artifactId>helidon-cli-harness</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.helidon.build-tools</groupId>
```

Comment:
Do we need this to parse the pom of the apptypes for the current "catalog" implementation ?

Reply:
You mean `helidon-utils`? Lots of things used from there: CLI logging, read/write POM, ANSI console classes, Maven utils, etc.

Reply:
Hmm, ok. I thought that the logging and ansi stuff was already added by Brian in cli-impl previously. We need to implement the catalog and start getting rid of the maven API dependencies.